Tests/FluentAssertions.Specs/Execution/AssertionScope.ContextDataSpecs.cs

```diff
@@ -41,9 +41,11 @@ public void Get_default_value_when_nullable_value_is_null()
     {
         // Arrange
         var scope = new AssertionScope();
-#pragma warning disable IDE0004 // Remove Unnecessary Cast
+
+        // ReSharper disable once RedundantCast
+        #pragma warning disable IDE0004
```

Comment:
Not at my dev machine right now, but I recall this triggers [IDE0055](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0055) about `#pragma` being indented.

Alternatively assign the `null` to a variable and we can git rid of both `#pragma` and `// ReSharper`
```
int? someValue = null;
scope.AddNonReportable("SomeKey", someValue);
```